src/select.js

```diff
@@ -901,6 +901,7 @@
             $select.refresh(attrs.refresh);
           });
 
+          $select.refreshDelay = uiSelectConfig.refreshDelay;
```

Comment:
here should be
```js
if(!attrs.hasOwnProperty('refreshDelay')) {
  $select.refreshDelay = uiSelectConfig.refreshDelay;
} else {
   attrs.$observe('refreshDelay', function() {/*etc*/})
}
```
because that's how `$observe` function works. It don't call a handler when attribute is doesn't exsits and we have to set  default value manually